packages/grpc-js/test/test-call-credentials.ts

```diff
@@ -147,12 +138,10 @@ describe('CallCredentials', () => {
       await Promise.all(
         testCases.map(async testCase => {
           const { credentials, expected } = testCase;
-          let metadata: Metadata;
-          try {
```

Comment:
@murgatroid99  this is another material change; I removed the redundant try/catch (since the catch block just rethrows the caught error).

This was causing a lint error and seems redundant anyway.